x/gamm/simulation/sim_msgs.go

```diff
@@ -355,7 +355,7 @@ func RandomExitSwapShareAmountIn(k keeper.Keeper, sim *simtypes.SimCtx, ctx sdk.
 		PoolId:            pool_id,
 		TokenOutDenom:     tokenOut.Denom,
 		ShareInAmount:     gammShares.Amount.Quo(sdk.NewInt(2)),
-		TokenOutMinAmount: tokenOut.Amount,
+		TokenOutMinAmount: tokenOut.Amount.Quo(sdk.NewInt(2)),
```

Comment:
The simulator was failing because of this. I think the calculations above are making it flaky so this change is made to unblock v12. Once the TODO above is addressed, this should be as well